lib/client-assets.php

```diff
@@ -365,6 +365,41 @@ function gutenberg_get_post_to_edit( $post_id ) {
 }
 
 /**
+ * Handles the enqueueing of block scripts and styles that are common to both
+ * the editor and the front-end.
+ *
+ * Note: This function must remain *before*
+ * `gutenberg_editor_scripts_and_styles` so that editor-specific stylesheets
+ * are loaded last.
+ *
+ * @since 0.3.0
+ */
+function gutenberg_common_scripts_and_styles() {
+	// Enqueue basic styles built out of Gutenberg through `npm build`.
+	wp_enqueue_style( 'wp-blocks' );
+
+	// Enqueue block styles built through plugins.  This lives in a separate
+	// action for a couple of reasons: (1) we want to load these assets
+	// (usually stylesheets) in *both* frontend and editor contexts, and (2)
+	// one day we may need to be smarter about whether assets are included
+	// based on whether blocks are actually present on the page.
```

Comment:
This should be a multi-line comment, e.g.

```
/*
 * ddd
 */
```

(with one asterisk on the first line)